pyproject.toml

```diff
@@ -27,9 +26,7 @@ lint.per-file-ignores = {"*tests*" = [
     "INP001",
     "T201",
 ]}
-lint.select = [
-    "ALL",
-]
+lint.select = ["ALL"]
```

Comment:
Why are you changing this? It was consistent throughout the file

Reply:
Sorry, I had an overzealous linter. Should be fixed now